resources/views/rfc.blade.php

```diff
@@ -2,6 +2,16 @@
     $metaImageUrl = action(\App\Http\Controllers\RfcMetaImageController::class, $rfc);
 @endphp
 
+@push('styles')
+    <link rel="stylesheet" href="https://unpkg.com/easymde/dist/easymde.min.css">
```

Comment:
Not sure yet if there is a better way to add this dependencies, I tried just installing the package via npm, but I wasn't able to initialize it.

Reply:
Looks fine to me, but I'm a total frontend noob. So if anyone has a better suggestion they can let us know :)